shift/models/shift_shift.py

```diff
@@ -29,8 +30,8 @@ def _get_selection_status(self):
     def _get_color_mapping(self, state):
         return {
             "draft": 0,
-            "open": 1,
-            "done": 5,
+            "open": 0,
+            "done": 10,
```

Comment:
Just a question on why changing this.

Reply:
Because the color code changes in Odoo 16. Don't know why they did, but with this change I get the same colors as in 12